src/v2/Utils/WorkflowEngine.tsx

```diff
@@ -73,4 +75,12 @@ export class WorkflowEngine {
   end() {
     return this.workflow.length - 1
   }
+
+  reset() {
+    this.moves = 0
+    this.index = 0
+    this.workflow = this.__workflow__
+
+    return this.current()
+  }
```

Comment:
Added this so that we could reset the workflow which isn't really useful for users but is useful for us when working on/testing the thing.